controllers/operator_controller.go

```diff
@@ -118,9 +109,9 @@ func (r *OperatorReconciler) reconcile(ctx context.Context, op *operatorsv1alpha
 	var message = "resolution was successful"
 
 	// run resolution
-	solution, err := r.resolver.Resolve(ctx)
+	solution, err := r.Resolver.Resolve(ctx)
 	if err != nil {
-		status = metav1.ConditionTrue
+		status = metav1.ConditionFalse
```

Comment:
Shouldn't this remain `ConditionTrue` as resolution _did_ fail?

Reply:
`ConditionTrue` is saying `Ready: True`.  Since resolution failed, shouldn't we be saying `Ready: false`?

Reply:
As an aside, I think the structure of how these condition values gets set makes it difficult to reason about. I've fixed this in #107, so as soon as this change lands, I can layer in another PR that simplifies the `Reconcile` function and makes it more obvious how/when the ready condition is set.

Reply:
You're right. I was conflating "Resolution failed" with the type.

OK, I think the confusion is that it's not obvious this is for the "Ready" type, since `TypeReady` is not very local, it's down 30+ lines.

Reply:
^ Exactly - That problem will be resolved in #118.